docs/using/installing/index.rst

```diff
@@ -4,17 +4,21 @@
 Installing Flocker
 ==================
```

Comment:
Before we explain how each bit is installed, perhaps keep the intention of the diagram which was to give an overview. In the getting started guide this is done with:

```
      Flocker Client
      --------------

      +--------------------------------------------------------------------------+
      | Runs on your laptop                                                      |
      +--------------------------------------------------------------------------+
      | Uses application and deployment configuration files                      |
      +--------------------------------------------------------------------------+
      | Deploys containers to a cluster of servers                               |
      +--------------------------------------------------------------------------+

      Flocker Node
      ------------

      +--------------------------------------------------------------------------+
      | Runs on each server in a cluster                                         |
      +--------------------------------------------------------------------------+
      | Links, ports and volumes work across hosts                               |
      +--------------------------------------------------------------------------+
      | After deployment, containers can move around                             |
      +--------------------------------------------------------------------------+
```